lms/djangoapps/experiments/tests/test_views.py

```diff
@@ -176,7 +176,7 @@ def cross_domain_config(func):
         'ENABLE_CROSS_DOMAIN_CSRF_COOKIE': True
     })
     settings_decorator = override_settings(
-        CORS_ORIGIN_WHITELIST=['ecommerce.edx.org'],
+        CORS_ORIGIN_WHITELIST=['https://ecommerce.edx.org'],
```

Comment:
https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/openedx/core/djangoapps/cors_csrf/helpers.py#L51

Due to this change test is failing.
```(Pdb) referer_hostname
'ecommerce.edx.org'
(Pdb) getattr(settings, 'CORS_ORIGIN_WHITELIST', [])
['https://ecommerce.edx.org']
```

Reply:
Following can fix it but not aware with complete workflow.
```
referer = request.META.get('HTTP_REFERER')
referer in getattr(settings, 'CORS_ORIGIN_WHITELIST', [])
```

Reply:
No, more tests are failing with this.

Reply:
This makes sense to me, so now rather than just comparing domains we'll compare scheme and domain.  And we don't need version checking around this because the library will be updated in the same PR, correct?

Reply:
@feanil latest version needed schemes. I have already merged the the internal [PR](https://github.com/edx/edx-internal/pull/5369/files#diff-ac896c0f56904af50ac3386fd925dd5b1799193857a2f2d26ff86e7d40290930R509)

In this [PR](#28588) I have added condition to pick new list if version is `3.2.0`

Reply:
@feanil please guide me finding edge `CORS_ORIGIN_WHITELIST` link.